instrumentation/armeria/armeria-grpc-1.14/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/armeria/grpc/v1_14/ArmeriaGrpcTest.java

```diff
@@ -29,6 +30,15 @@ class ArmeriaGrpcTest {
   @RegisterExtension
   static final AgentInstrumentationExtension testing = AgentInstrumentationExtension.create();
 
+  private static final AttributeKey<Long> RPC_CLIENT_REQUEST_BODY_SIZE =
+      AttributeKey.longKey("rpc.client.request.body.size");
+  private static final AttributeKey<Long> RPC_CLIENT_RESPONSE_BODY_SIZE =
+      AttributeKey.longKey("rpc.client.response.body.size");
+  private static final AttributeKey<Long> RPC_SERVER_REQUEST_BODY_SIZE =
+      AttributeKey.longKey("rpc.server.request.body.size");
+  private static final AttributeKey<Long> RPC_SERVER_RESPONSE_BODY_SIZE =
+      AttributeKey.longKey("rpc.server.response.body.size");
```

Comment:
in tests you should use `RpcIncubatingAttributes`

Reply:
These attributes do not currently exist in `RpcIncubatingAttributes`, Maybe we should define them in [semantic-conventions](https://github.com/open-telemetry/semantic-conventions/)

Reply:
yes, you should try getting these included in the semantic attributes